go/pserver/service.go

```diff
@@ -9,8 +9,10 @@ import (
 // ElementType is the type of elements of a Parameter.
 type ElementType int
 
-var ErrAlreadyInitialized = errors.New("pserver already initialized")
-var ErrUninitialized = errors.New("pserver not fully initialized")
+const (
```

Comment:
Because if we are using
```go
var ErrAlreadyInitialized = errors.New("pserver already initialized")

func Foo(a int b *int) error {
  return ErrAlreadyInitialized
}
```
The RPC client can not compare the error:
```
err := client.Call("Service.Foo", 1, nil)
fmt.Println(err == ErrAlreadyInitialized)
// Output:
// false
```
This is happening because `err` is of type `error`, which is a interface.
[Interface comparison rule](https://golang.org/ref/spec):
> A value x of non-interface type X and a value t of interface type T are comparable when values of type X are comparable and X implements T. They are equal if t's dynamic type is identical to X and t's **dynamic value is equal to x**.

Since the `err` instance is deserialized by RPC, which is of different instance (dynamic value) than `ErrAlreadyInitialized`